libr/core/panels.c

```diff
@@ -741,8 +741,14 @@ static void activateCursor(RCore *core) {
 			|| !strcmp (cur->model->cmd, PANEL_CMD_REGISTERS)
 			|| !strcmp (cur->model->cmd, PANEL_CMD_DISASSEMBLY)
 			|| !strcmp (cur->model->cmd, PANEL_CMD_HEXDUMP)) {
+		if (cur->model->cache) {
+			(void)r_cons_yesno ('y', "Please turn off cache by \'&\' if you wish to use cursor.");
```

Comment:
you have r_cons_anykey() for that. those messages are not questions

Reply:
lol didn't know the API even though I tried to find the one just like that :P thanks